.github/workflows/test-ci.yml

```diff
@@ -119,3 +119,89 @@ jobs:
 
             -   name: ESLint
                 run: npm run lint
+
+    release_next:
+        name: Release @next
+        if: "!github.event.pull_request && (!contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, 'docs:'))"
```

Comment:
should be "only master" instead of "not PR". the workflow is also executed for renovate updates in other branches